docs/getting-started.md

```diff
@@ -4,26 +4,192 @@
 
 ### Building from source
 
-Prerequisites: The [Go](https://golang.org/doc/install) compiler **version 1.15** (we use
-go modules) or later and GNU make. The go bin path (usually `$HOME/go/bin`) should be
-in your system `$PATH`.
+#### Prerequisites
 
-Run `make install` to build the code. The `regen` binary serves as both the blockchain daemon, as well as
+**Minimum Hardware Requirements**
+
+- 8GB RAM
+- 4vCPUs
+- 200GB Disk space
+
+**Software Requirements** 
+
+Before beginning make sure you have the [Go](https://golang.org/doc/install) compiler **version 1.15** (we use
+go modules) or later and GNU make installed and add the go bin path (usually `$HOME/go/bin`) should be in your system `$PATH`. To learn more about adding commands to your path please visit [linuxize.com](https://linuxize.com/post/how-to-add-directory-to-path-in-linux/). We recommend using Ubuntu 18.04 or higher, the official images can be found at [Ubuntu.com](https://ubuntu.com/tutorials/install-ubuntu-desktop#1-overview)
+
+---
+
+#### Download and Install Regen Ledger
+
+Install git and build-essentials for ubuntu
+```sh
+sudo apt update
+```
+```sh
+sudo apt install git build-essential jq -y
+```
+
+Download go and extract tarball
+```sh
+wget https://dl.google.com/go/go1.15.2.linux-amd64.tar.gz
+```
+```sh
+tar -xvf go1.15.2.linux-amd64.tar.gz
+```
+```sh
+sudo mv go /usr/local
+```
+
+Add go to local path
+```sh
+echo "" >> ~/.profile
+```
+```sh
+echo 'export GOPATH=$HOME/go' >> ~/.profile
+```
+```sh
+echo 'export GOROOT=/usr/local/go' >> ~/.profile
+```
+```sh
+echo 'export GOBIN=$GOPATH/bin' >> ~/.profile
+```
+```sh
+echo 'export PATH=$PATH:/usr/local/go/bin:$GOBIN' >> ~/.profile
+```
+
+```sh
+. ~/.profile
+```
```

Comment:
Feels like a lot of lines for just getting someone to setup their gopath stuff. Can we consolidate these into one multiline code snippet?